lib/round.rb

```diff
@@ -0,0 +1,52 @@
+class Round
+    attr_reader :deck, :turns, :cards, :number_correct
+    
+    def initialize (deck)
+        @deck = deck
+        @cards = cards
+        @turns = []
+        @number_correct = 0
+        @correct_turns = @correct_turns
+        @category_turns = @category_turns
+
+    end
+
+    def current_card
+        deck.cards[0]
+    end
+
+    def take_turn(guess)
+        turn = Turn.new(guess, current_card)
+        @turns << turn
+        if guess == current_card.answer
```

Comment:
How about instead of `if guess == current_card.answer` we do `if turn.correct?`? Would that still work?